staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -651,6 +651,12 @@ const (
 	// https://kep.k8s.io/3998
 	// This is currently an alpha field.
```

Comment:
also beta?

Reply:
That's true. Actually, I modified the comment in the Beta Graduation PR (https://github.com/kubernetes/kubernetes/pull/126064). But, as we discussed in https://github.com/kubernetes/kubernetes/pull/126064#issuecomment-2232586393, we will consolidate all Graduation PRs into a single PR, and then we can check all changes in the single PR.